tests/rules/selectors-start-with-select.test.ts

```diff
@@ -0,0 +1,56 @@
+import { stripIndent } from 'common-tags'
+import { fromFixture } from 'eslint-etc'
+import path from 'path'
+import rule, {
+  messageId,
+} from '../../src/rules/store/selectors-start-with-select'
+import { ruleTester } from '../utils'
+
+ruleTester().run(path.parse(__filename).name, rule, {
+  valid: [
+    `
+    interface AppState {
+      feature: FeatureState;
+    }
```

Comment:
Since this rule doesn't deal with type checker, coudn't we just drop this and other interfaces in this file to avoid unnecessary distractions?

Reply:
I agree